_includes/indexes/hnsw-intro.mdx

```diff
@@ -0,0 +1 @@
+[Hierarchical Navigable Small World (HNSW) indexes](/developers/weaviate/concepts/indexing/hnsw-indexes) are high-performance, in-memory indexes. HNSW indexes scale well; vector searches are fast, even for very large data sets.
```

Comment:
Might be worth noting how they scale. (I believe logarithmically, in O(log n) - using CS terms.)